unity-renderer/Assets/DCLServices/WearablesCatalogService/WearablesWebInterfaceBridge.cs

```diff
@@ -0,0 +1,13 @@
+﻿using DCL.Interface;
+
+namespace DCLServices.WearablesCatalogService
+{
+    public class WearablesWebInterfaceBridge
```

Comment:
I dont find a real purpose of this class at this point. Perhaps after you make the test coverage

Reply:
It was exactly for ease the tests (they're not still implemented). My idea was to mock this class in the tests.